solr/solrj/src/test/org/apache/solr/client/solrj/impl/CloudHttp2SolrClientTest.java

```diff
@@ -268,35 +258,90 @@ public void testAliasHandling() throws Exception {
   }
 
   @Test
-  @LogLevel("org.apache.solr.servlet.HttpSolrCall=INFO")
+  @LogLevel("org.apache.solr.servlet.HttpSolrCall=DEBUG")
   public void testHttpCspPerf() throws Exception {
 
-    String COLLECTION = "TEST_COLLECTION";
-    CollectionAdminRequest.createCollection(COLLECTION, "conf", 2, 1)
-        .process(cluster.getSolrClient());
-    cluster.waitForActiveCollection(COLLECTION, 2, 2);
+    performTest(false, "HTTPCSPTEST",
+            1,
+            10,
+            //1 create collection, 2 /admin/core , 1 LISTALIASES, 1 CLUSTERSTATUS, 10 CLUSTERSTATUS with coll param
+            15,
+            15);
 
-    SolrInputDocument doc = new SolrInputDocument("id", "1", "title_s", "my doc");
-    httpBasedCloudSolrClient.add(COLLECTION, doc);
-    httpBasedCloudSolrClient.commit(COLLECTION);
+  }
+  @Test
+  @LogLevel("org.apache.solr.servlet.HttpSolrCall=DEBUG")
+  public void testZkCspPerf() throws Exception {
```

Comment:
Why test with ZkCsp using an HTTP logging approach?  ZkCsp does not make HTTP calls.  If we could monitor TCP ZK connections somehow, *that* would be interesting.  Such an approach would be doable with distributed tracing tech or SolrZkClient metrics (even easier) but I don't want to suggest expanding the scope/effort of this PR too much.

Reply:
Ok I guess it makes sense to remove this test